pyproject.toml

```diff
@@ -61,6 +61,8 @@ testpaths = [
 
 [tool.isort]
 profile = "black"
+line_length = 88
+wrap_length = 88
```

Comment:
I know! But I keep getting these otherwise!

```diff
-from src.jbi.whiteboard_actions import default_with_assignee_and_status as action
+from src.jbi.whiteboard_actions import \
+    default_with_assignee_and_status as action
```